_data/people.yml

```diff
@@ -108,6 +108,11 @@
    email: jay.vakil@colorado.edu
    pic: jay.jpg
 
+- name: Yutong Zhang
```

Comment:
Be sure to fix the spacing and make it EXACTLY the same as others. Otherwise, the website breaks :-)

Reply:
I was so confused why the preview was not working. I did not notice the 1 whitespace at all.
Thanks for pointing that out. It is now fixed.

Reply:
Yeah it is soooo stupid -- but it is what it is :-)